### PR TITLE
feat: adds artifacthub  Verified Publisher

### PR DESCRIPTION
**Pull request by nilekhc**
Signed-off-by: Nilekh Chaudhari <1626598+nilekhc@users.noreply.github.com>

### CODE REVIEW

**Comment by nilekhc** [comment]
cc @maxsmythe @sozercan @ritazh

File: artifacthub/artifacthub-repo.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+repositoryID: 2b7422af-5f40-489d-9f89-dfb358f9a485
+owners:
+  - name: nilekh
+    email: nilekhc@gmail.com
+  - name: sozercan
+    email: sozercan@gmail.com
```
Comment (sozercan) [inline artifacthub/artifacthub-repo.yml:6]: @ritazh @maxsmythe do you want to add your email here?

**Review by sozercan — APPROVED** [review]
LGTM

File: artifacthub/artifacthub-repo.yml (hunk 1 above)
Comment (nilekhc) [reply artifacthub/artifacthub-repo.yml:6]: You need an account on AH for `name`.

**Comment by sozercan** [comment]
@nilekhc looks like generate is failing

**Comment by nilekhc** [comment]
> @nilekhc looks like generate is failing

Yup. Fixed it.

File: artifacthub/artifacthub-repo.yml (hunk 1 above)
Comment (nilekhc) [reply artifacthub/artifacthub-repo.yml:6]: @sozercan 
```
Please make sure the email used in the metadata file matches with the one you use in Artifact Hub.
```

Let me know if your email is correct.

Comment (nilekhc) [reply artifacthub/artifacthub-repo.yml:6]: Looks like you have the correct email ID @sozercan. Thanks for confirming.

**Review by maxsmythe — APPROVED** [review]
LGTM

**Pull request merged**